MAINTAINERS.md

```diff
@@ -0,0 +1,154 @@
+> ~~ feat(karma-runner): resolved local karma and ng version ~~
+
+❌ Format issues:
+> ~~ feat(Html reporter) Add html reporter ~~ // missing :
+> ~~ feat: Add html reporter (#2622) ~~ // missing subject
+
+❌ Scope too vague
+> ~~feat(stryker): changed option~~
+
+
+It is important to note that these message are **not necessary when contributing a PR**, only when **squash merging them** (we only care about commit messages on the master branch).
+
+Once more: **Only use squash merge to the master branch**.
+
+### Commenting on Issues and Reviewing Pull Requests
+**All maintainers should be courteous and kind.** Thank the external contributor for the pull request, even if it is not merged. If the pull request has been opened (and subsequently closed) without discussion in a corresponding issue, let them know that by creating an issue first, they could have saved wasted effort. _Clearly and objectively_ explain the reasoning for rejecting any PR.
+
+If you need more information on an issue, nicely ask the user to provide it. Remind them to use the issue/PR templates if they have not. If the user never gets around to it, the "stalebot" will close it eventually anyhow.
+
+#### Reviewing Code
+Use GitHub's code review features. Requesting a review from another maintainer _may or may not_ actually result in a review; don't wait on it. If the PR cannot move forward without input from a certain maintainer, _assign them to the PR_. If you’re waiting on someone, send them a direct message on Slack.
+
+#### The Part About Jerks
+There will be jerks.
+
+#### Rude or Entitled People
+These are users who feel the Stryker-net project and its maintainers _owe them_ time or support. This is incorrect.
+
+However, this behavior is often indicative of someone who is "new" to open source. Many just don't know better. It is not your _responsibility_ to educate them (again, you owe them nothing).
+
+Here are some suggestions:
+1. If u mad, wait 20 minutes before writing a comment.
+1. "Kill them with kindness". Explain how they are presenting themselves; maybe link to a good article or two about it.
+1. Don't make it about "users vs. maintainers". Treat them like a potential future maintainer.
+1. Avoid adding to the drama. You could try to reach out privately; email may be in their GitHub profile. You will likely never hear from that individual again (problem solved)
+1. If an issue is getting out of control, lock it.
+1. If someone is _repeatedly_ rude and does not correct their mistakes, you may ban them from participating in the `stryker-mutator` org. If you do not have permission to do so, contact someone who does (an "owner").
+
+### When do we release?
+We release when we feel like it. However, we don’t want to overwhelm users with new versions. That’s why we’ve decided to release major versions a maximum of 2 times a year. Furthermore, we might decide to group features together, in order to not release multiple minor versions in a short period of time.
```

Comment:
I suggest having release policy separately, IMO it's about something else than the rest of the doc. But even if you keep this here, it would be useful to elaborate on what is major release, minor release, maybe patches or hotfixes, something like this. 

Lastly, I don't understand the argument about overwhelming users with new versions. As long as the changelog is clear and I can find what version has what, I'm okay :)

Reply:
@richardwerkman this is exactly the kind of thing I want to carefully look at and not just copy from strykerjs. I don't agree with this policy of a maximum number of major version releases.

Reply:
Well with our current pace we will never hit that 2 major releases a year 🙈 But I'm fine with removing this. If we see the need for a third (or even more) major releases in a year I don't see why we should hold back.